sdk/src/org.graalvm.nativeimage/src/org/graalvm/nativeimage/hosted/RuntimeClassInitialization.java

```diff
@@ -150,6 +150,8 @@ public static void initializeAtRunTime(String... packages) {
     /**
      * Registers all classes in provided packages as eagerly initialized during image-build time.
      * <p>
+     * Passing {@code ""} as a package, registers all packages and classes for initialization at build time.
```

Comment:
I'd be inclined to add a disclaimer like "Use at your own risk as this can have unintended side-effects.". Even if Quarkus leverage this, we shouldn't really recommend this to other users.

Reply:
I added "This might have unintended side-effects and should thus be used with great caution."